src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx

```diff
@@ -66,8 +77,30 @@ export const MyCollectionArtworkSidebarMetadataFragmentContainer = createFragmen
   }
 )
 
-const MetadataField = ({ label, value }) => {
+export const MetadataField = ({
+  label,
+  value,
+  truncateLimit = 0,
+}: {
+  label: string
+  value?: string | null
+  truncateLimit?: number
+}) => {
   const emptyValue = "----"
+  const [expanded, setExpanded] = useState(false)
+  const [modalOpen, setModalOpen] = useState(false)
+  const isMobile = __internal__useMatchMedia(themeProps.mediaQueries.xs)
```

Comment:
**issue:** `__internal__useMatchMedia` is marked as internal and we should avoid using it ([Note why it should be avoided](https://github.com/artsy/force/blob/main/src/Utils/Hooks/useMatchMedia.ts#L4)). I think it should be possible to use `Media` instead.

Reply:
@damassi what do you think?

Reply:
Thanks @olerichter00  sorry I missed this comment. In this case I am using isMobile to conditionally toggle in the `toggle` function. It is not directly used for UI rendering or to manipulate responsiveness. I suppose that should be fine.